example/gluon/image_classification.py

```diff
@@ -255,7 +255,10 @@ def main():
         data = mx.sym.var('data')
         out = net(data)
         softmax = mx.sym.SoftmaxOutput(out, name='softmax')
-        mod = mx.mod.Module(softmax, context=[mx.gpu(i) for i in range(num_gpus)] if num_gpus > 0 else [mx.cpu()])
```

Comment:
why not change to
`mod = mx.mod.Module(softmax, context=context)` directly?

Reply:
@piiswrong , thanks for review and you are right, commit revised. Thanks.